oasis/models/iforest.py

```diff
@@ -7,32 +7,25 @@
 import pandas as pd
 import datetime
 import random
-import yaml
-import os
-from threading import Thread
 from oasis.datasource import PrometheusAPI, PrometheusQuery, Metrics
 from sklearn.ensemble import IsolationForest
-from threading import Event, Lock, Timer
+from threading import Timer, Thread
```

Comment:
Can we get rid of `threading`? This may not be safe in tornado.

Try `tornado.ioloop.PeriodicCallback`?